ui/src/pages/backoffice/Series/SeriesDetails/components/SeriesRelations/RelationSequence/RelationSequence.js

```diff
@@ -0,0 +1,102 @@
+import { Error, Loader } from '@components';
+import { SeriesLanguages } from '@components/Series';
+import { InfoMessage, SeriesDetailsLink } from '@pages/backoffice';
+import { Divider, Grid } from 'semantic-ui-react';
+import { RelationSequenceModal } from '../RelationSequence';
+import {
+  RelationRemover,
+  ExistingRelations,
+} from '@pages/backoffice/components/Relations';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+
+export default class RelationSequence extends Component {
+  viewDetails = ({ row }) => {
+    return (
+      <SeriesDetailsLink seriesPid={row.pid}>{row.title}</SeriesDetailsLink>
+    );
+  };
+
+  removeHandler = ({ row }) => {
+    const { seriesDetails } = this.props;
+
+    if (!isEmpty(seriesDetails)) {
+      return (
+        <RelationRemover
+          referer={seriesDetails}
+          related={row}
+          buttonContent={'Remove relation'}
+        />
+      );
+    }
+  };
+
+  languagesFormatter = ({ row }) => {
+    return <SeriesLanguages metadata={row} />;
+  };
+
+  render() {
+    const { relations, showMaxRows, isLoading, error } = this.props;
+    const next = relations['next'] || [];
+    const previous = relations['previous'] || [];
+
+    const columns = [
+      { title: 'Title', field: 'title', formatter: this.viewDetails },
+      {
+        title: 'Languages',
+        field: 'languages',
+        formatter: this.languagesFormatter,
+      },
+      { title: 'Actions', field: '', formatter: this.removeHandler },
+    ];
+
+    return (
+      <Loader isLoading={isLoading}>
+        <Error error={error}>
+          <RelationSequenceModal relationType={'sequence'} />
+          <Grid columns={2} relaxed="very">
+            <Grid.Column width={8}>
+              <Divider horizontal> Predecessors of this serial </Divider>
+              <ExistingRelations
+                rows={previous}
+                showMaxRows={showMaxRows}
+                columns={columns}
+                emptyMessage={
+                  <InfoMessage
+                    header={'No predecessors'}
+                    content={'Use the button above to sequence relations.'}
+                  />
+                }
+              />
+            </Grid.Column>
+            <Grid.Column width={8}>
+              <Divider horizontal> Continuations of this serial </Divider>
+              <ExistingRelations
+                rows={next}
+                showMaxRows={showMaxRows}
+                columns={columns}
+                emptyMessage={
+                  <InfoMessage
+                    header={'No continuations'}
+                    content={'Use the button above to sequence relations.'}
```

Comment:
Sequence is not verb, maybe something like "Use the button above to create a sequential relation"

Reply:
👀  thanks I ate the word